src/rdb/rdb_raft.c

```diff
@@ -1118,16 +1115,23 @@ rdb_raft_update_node(struct rdb *db, uint64_t index, raft_entry_t *entry)
 	return rc;
 }
 
+/* See rdb_raft_log_offer_single. */
+#define RDB_RAFT_ENTRY_NVOPS 2
```

Comment:
Sorry if it is evident / should be known already. This is the last file of the set I'm looking at and may be losing steam. :)

For rdb_raft_log_offer_single() this value (2) corresponds to the rdb_lc_update() call and the rdb_mc_update() call in that function? Or, does it refer to the (potential) n=2 keys/values fed into rdb_lc_update()